portals/application/languages/es_ES/es_ES.po

```diff
@@ -2820,6 +2781,9 @@ msgstr "Loc. Error"
 msgid "Success locution id"
 msgstr "Loc. Éxito"
 
+msgid "Timeout"
+msgstr "Timeout"
```

Comment:
That was detected and generated as new, but I have no idea why it's the first time it appears tbh